lua/terrortown/lang/tr.lua

```diff
@@ -820,17 +820,17 @@ L.aw_tod2_text = "raundun başlamasından hemen sonra öldü."
 
 -- New and modified pieces of text are placed below this point, marked with the
 -- version in which they were added, to make updating translations easier.
-
+-- New comment format for this language file is YY-MM-DD
```

Comment:
no, we use YYYY-MM-DD. Our language tool will revert your comment changes anyway

Reply:
Ok no problem 👌

Reply:
you can keep it that way and I run the language tool on your PR. So you don't have to manually edit it